<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreCrearNomina</class>
 <widget class="QDialog" name="PreCrearNomina">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Nómina</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/crearnomina.png</normaloff>:/crearnomina.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/atras.png</normaloff>:/atras.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Elige un Año...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbAno"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Elige una Nómina...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbMes"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>71</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Horas Extras Diurnas...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etHed">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Horas Extras Nocturnas...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etHen">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Horas Extras Festivas...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etHef">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Plus Tóxicos...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etToxicos">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Cte. Prorrateo...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etProrrateo">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>71</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dias Totales...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etDiasTotales">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Antiguedad...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etAntiguedad">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Plus Asistencia...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etAsistencias">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Plus Transporte...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etTransporte">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Turnucidad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etTurnicidad">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,1">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Retención I.R.P.F.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="etIrpf">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
